cli/application/src/main/java/com/buschmais/jqassistant/commandline/task/AbstractTask.java

```diff
@@ -7,11 +7,13 @@
 import com.buschmais.jqassistant.commandline.Task;
 import com.buschmais.jqassistant.core.rule.api.RuleHelper;
 import com.buschmais.jqassistant.core.runtime.api.plugin.PluginRepository;
+import com.buschmais.jqassistant.core.shared.annotation.Description;
 import com.buschmais.jqassistant.core.store.api.StoreFactory;
 
 import org.apache.commons.cli.CommandLine;
 import org.apache.commons.cli.Option;
 
+@Description("Abstract base implementation for analysis mojos.")
```

Comment:
There should be no description for the abstract base class as it should never be printed as part of the help task